### ISSUE TITLE
Use of the terms "user story"

### ISSUE THREAD

**Issue by dboudreau** [w3c/wai-quick-start#314]
After having given it some thought, I am uncomfortable with using the terms "user stories" in our quick tips to link to the content from the "How people use the Web" resource. I would be in favor of changing "User story" to "Persona", which I think is much closer to what we're conveying, here. While these are stories about users, that particular term has been deviated from its original meaning in Agile to mean the description of a requirement, based on the expectation of a person using the site/app/software/etc.

In most designers' and developers' minds nowadays, the expectation of a user story is likely to be related to a very high-level definition of a requirement or a feature, in the form of "as a [x], I want/need [y], so that I can [z]". This is not what we're providing here. I think it's misleading, and I think we'D be better of avoiding that confusion if we can.

So this a global proposal to replace "User story" by "Persona", on every tip that links to one:

## Designing
- https://www.w3.org/WAI/gettingstarted/tips/designing.html#provide-sufficient-contrast-between-foreground-and-background
- https://www.w3.org/WAI/gettingstarted/tips/designing.html#dont-use-color-alone-to-convey-information
- https://www.w3.org/WAI/gettingstarted/tips/designing.html#ensure-that-interactive-elements-are-easy-to-identify
- https://www.w3.org/WAI/gettingstarted/tips/designing.html#provide-clear-and-consistent-navigation-options
- https://www.w3.org/WAI/gettingstarted/tips/designing.html#ensure-that-form-elements-include-clearly-associated-labels
- https://www.w3.org/WAI/gettingstarted/tips/designing.html#provide-easily-identifiable-feedback
- https://www.w3.org/WAI/gettingstarted/tips/designing.html#use-headings-and-spacing-to-group-related-content
- https://www.w3.org/WAI/gettingstarted/tips/designing.html#create-designs-for-different-viewport-sizes
- https://www.w3.org/WAI/gettingstarted/tips/designing.html#include-image-and-media-alternatives-in-your-design
## Developing
- https://www.w3.org/WAI/gettingstarted/tips/developing.html#include-alternative-text-for-images
- https://www.w3.org/WAI/gettingstarted/tips/developing.html#use-mark-up-to-convey-meaning-and-structure
- https://www.w3.org/WAI/gettingstarted/tips/developing.html#help-users-avoid-and-correct-mistakes
- https://www.w3.org/WAI/gettingstarted/tips/developing.html#write-code-that-adapts-to-the-users-technology
- https://www.w3.org/WAI/gettingstarted/tips/developing.html#ensure-that-all-interactive-elements-are-keyboard-accessible
## Writing
- https://www.w3.org/WAI/gettingstarted/tips/writing.html#use-headings-to-convey-meaning-and-structure
- https://www.w3.org/WAI/gettingstarted/tips/writing.html#write-meaningful-text-alternatives-for-images
- https://www.w3.org/WAI/gettingstarted/tips/writing.html#create-transcripts-and-captions-for-multimedia
- https://www.w3.org/WAI/gettingstarted/tips/writing.html#provide-clear-instructions
- https://www.w3.org/WAI/gettingstarted/tips/writing.html#keep-content-clear-and-concise

**Comment by nitedog** [comment]
Hi Denis,

Good point, I agree that "user story" is now more frequently used to mean something slightly different (at least in format, I think we do provide a lot of the content to actually allow these to become user stories).

On the other hand, I recall we had decided against "persona" because it is too jargony, and because the target page is called "stories of web users" - we wanted something to reflect that title in a simple way.

How about something like "user example(s)" or "user outline(s)" or such? Or reverting it to (the slightly longer) "examples of users" or such? AFAIK this phrase is only used as a list heading, so may be sufficient.

**Comment by dboudreau** [comment]
That's good feedback, thanks. I can understand how "persona" could also be considered jargony. I'll give it some thought as well, but I think I'd be ok with something like "examples" or "outlines" but I'd like to avoid the use of the word "user" if possible. I'd rather we refer to them as people, not users.

**Comment by nitedog** [comment]
Good point, I agree. Just needs contextualization because "examples of people" is just to broad. Maybe "examples of people using the web", but that is also rather long. Happy brainstorming... ;-)

**Comment by dboudreau** [comment]
After some thought, I'm leaning towards using "User Scenario" for this, AND using "User Story" to actually create a real, agile-like user story for each tip, such as "as a user with [x] disability, I want/need [y], so that I can [z]". Thoughts?

**Comment by AndrewArch** [comment]
Even 'personas' are going out of vogue these days to in the agile development world (not that everyone is going that way yet). 'Scenario' works for me. I do like the idea of providing agile users stories as examples at some stage / somewhere for people working in that way too to show them how to bring accessibility requirements into the backlog.

**Comment by nitedog** [comment]
I like "User Scenario" as well. The term "scenario" may have some specific implications in some contexts but I think it is a sufficiently generic and common term to use in a general sense.

Besides that, I really do *not* recommend recreating the content (user stories) of the target resource (or creating new content). This seems like a huge scope-creep - I would really not under-estimate the work involved. Maybe better to complete the currently open task of completing the tips, then come back to this issue later on - either by updating the target resource or by adding new content to this resource.

**Comment by dboudreau** [comment]
@nitedog I hear you. I'm not really concerned with scope creep for something like this, though. I know we tend to discuss this for a long time, but those user stories would only be one sentence long, and for the most part. pretty straightforward. As an example, the user story for the first tip in design could be something along the lines of: "As a person who is color blind, I want sufficient contrasts between foreground and background colors, so I can easily perceive content". Just a little something to contextualize the tips, and help people understand why they should care.

**Comment by sharronrush** [comment]
+1 to User Scenario, thanks Denis for good thinking on this.

**Comment by nitedog** [comment]
interesting idea @dboudreau - might be worth a try if all tips can be reduced to such a simple one-liner.
